File: Corte1/C++/Persona.cpp
#include <iostream>
#include <string>
using namespace std;

class Usuario {
public:
    string nombreCompleto;
    int años;

    // metodo para mostrar datos
    void imprimir() {
        cout << "Nombre = " << nombreCompleto << "\n";
        cout << "Edad = " << años << "\n";
    }
};

int main() {

    // creacion de objetos
    Usuario personaA;
    Usuario personaB;

    // asignacion de valores
    personaA.nombreCompleto = "Sebastian";
    personaA.años = 20;

    personaB.nombreCompleto = "Laura";
    personaB.años = 16;

    // validacion de edad
    if (personaA.años >= 18) {
        cout << "Persona A es mayor de edad\n";
    } else {
        cout << "Persona A es menor de edad\n";
    }

    cout << "\n";

    if (personaB.años >= 18) {
        cout << "Persona B es mayor de edad\n";
    } else {
        cout << "Persona B es menor de edad\n";
    }

    cout << "\nDatos persona A:\n";
    personaA.imprimir();

    cout << "\nDatos persona B:\n";
    personaB.imprimir();

    return 0;
}

File: Corte1/Python/Persona.py
class Individuo:
    def __init__(self):
        self.nombre = ""
        self.edad = 0

    def mostrar(self):
        print("Nombre =", self.nombre)
        print("Edad =", self.edad)


# creacion de objetos
ind1 = Individuo()
ind2 = Individuo()

# asignacion de datos
ind1.nombre = "Mateo"
ind1.edad = 25

ind2.nombre = "Sara"
ind2.edad = 17

# validacion de edad
if ind1.edad >= 18:
    print("Individuo 1 es mayor de edad")
else:
    print("Individuo 1 es menor de edad")

print()

if ind2.edad >= 18:
    print("Individuo 2 es mayor de edad")
else:
    print("Individuo 2 es menor de edad")

print("\nDatos individuo 1:")
ind1.mostrar()

print("\nDatos individuo 2:")
ind2.mostrar()